scripts/globals/gobbiemysterybox.lua

```diff
@@ -71,22 +71,22 @@ local anniversaryItems =
 }
 local gobbieJunk =
 {
-    507,
-    508,
-    510,
-    511,
-    566,
-    568,
-    1239,
-    1868,
-    2542,
-    2543,
-    4539,
-    4543,
-    9777,
-    15203,
-    18180,
-    19220,
+    507,   -- Goblin Mail
+    508,   -- Goblin Helm
+    510,   -- Goblin Armor
+    511,   -- Goblin Mask
+    566,   -- Goblin Cup
+    568,   -- Goblin Die
+    1239,  -- Goblin Doll
+    1868,  -- Goblin Coif Cutting
+    2542,  -- Goblin Mess Tin
+    2543,  -- Goblin Weel
+    4539,  -- Goblin Pie
+    4543,  -- Goblin Mushpot
+    9777,  -- Goblin Offering
+    15203, -- Goblin Coif
+    18180, -- Goblin Grenade
+    19220, -- Goblin Cracker
```

Comment:
After literally 4 full stacks of 99 keys (396 trades total) that coudl only produce rare flagged stuff I already had (abjuration keys) I got:

71 goblin mess tins,
106 goblin pies
102 hobgoblin pies
48 hobgoblin chocolates
40 goblin chocolates
14 goblin bread
9 hobgoblin bread
6 goblin weel
and nothing else. not a single armor helm or mask.

Which suggestes they are weighted and makes me question if some of the other goblin items can even happen at all, or if maybe abjur keys don't pull from the same list.

Reply:
I actually do this for the mess tins, they desynth to darksteel :)

Reply:
Admittedly it's been a while since this list was compiled and I'm pretty sure I just grabbed whatever "Goblin" items that had the "Obtainable from Goblin Box" flag. It's likely some items are missing and some maybe shouldn't be there, along with possible weighting.

Reply:
oh. well, every time my mules build up to many keys I can get us more data xD I keep a pile of abjurs I just pass to whichever one has too many keys to make my darksteel factory go brrrr so I'll keep tracking them for later use.

Reply:
So I went back and looked over those items and some of them were themselves Rare flagged so I'm not sure what criteria was used when originally selecting them. I went ahead and changed the list to reflect your findings, minus any possible weighting.